packages/evolution-frontend/src/components/admin/validations/InterviewMap.tsx

```
@@ -1,150 +1,504 @@
-import React, { useRef, useState } from 'react';
+
+import React, { useRef, useState, useEffect, useMemo } from 'react';
 import config from 'evolution-common/lib/config/project.config';
-import { withTranslation } from 'react-i18next';
-import DeckGL, { DeckGLRef } from '@deck.gl/react';
-import { WebMercatorViewport, MapViewState } from '@deck.gl/core';
-import { TileLayer } from '@deck.gl/geo-layers';
-import { IconLayer, BitmapLayer, PathLayer } from '@deck.gl/layers';
-import AnimatedArrowPathExtension from './AnimatedArrowPathLayerExtension';
+import { useTranslation } from 'react-i18next';
+import Map, { MapRef, useControl, Marker } from 'react-map-gl/maplibre';
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)